Lang/Lang.analyzer.lex

```diff
@@ -0,0 +1,87 @@
+%namespace ChineseObjects.Lang
+%scannertype LangScanner
+%visibility internal
+%tokentype Token
+
+%option stack, minimize, parser, verbose, persistbuffer, noembedbuffers
+
+Eol             (\r\n?|\n)
+NotWh           [^ \t\r\n]
+Space           [ \t]
+IntegerLiteral  [0-9]+
+RealLiteral     [0-9]+\.[0-9]
```

Comment:
This doesn't look correct. I would definitely like it to match `1.23`, also, probably, things like `1.` and `.9`